ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueConsumption.scala

```diff
@@ -371,7 +379,11 @@ object KeyValueConsumption {
               Status.of(
                 Code.ABORTED.value,
                 reason,
-                Seq.empty,
+                Seq(
+                  AnyProto.pack[ErrorInfo](
+                    ErrorInfo(metadata = Map(GrpcStatuses.DefiniteAnswerKey -> "false"))
```

Comment:
Same here as above. I.e., `KeyValueConsumption` is called on the read path, hence, it must not override the conclusion made by the committer (and communicated via the rejection entry).